api/anri/apps/products/views.py

```diff
@@ -17,9 +18,10 @@ class ProductViewSet(viewsets.ModelViewSet):
     queryset = Product.objects.order_by("created")
     serializer_class = ProductSerializer
     pagination_class = ProductPagination
+
+    filter_backends = (filters.SearchFilter, django_filters.rest_framework.DjangoFilterBackend)
```

Comment:
from rest_framework.filters import SearchFilter
from django_filters.rest_framework import DjangoFilterBackend

Reply:
I'll fix it up to meeting

Reply:
Done